CHANGELOG.md

```diff
@@ -4,9 +4,20 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Packaging
+
+- Minimum rust version was bumped to `1.41.0`
+
+### Removed
+
+- `create_clipboards` API on Wayland
+- Example for Wayland clipboard
```

Comment:
Why was the example removed entirely? Just because it was too complicated to maintain?

Reply:
@chrisduerr because it'll be the exact copy of the one in `smithay-clipboard`, if you want it here, I can just copy that code, idk. The code https://github.com/Smithay/smithay-clipboard/blob/master/examples/clipboard.rs . So I really not sure that it should be maintained in 2 places.

Reply:
Fair enough, it's not like there are some massive differences between the platforms anyways.